sdk/app_cpu1/common/sys/scheduler.c

```diff
@@ -1,13 +1,16 @@
 #include "sys/scheduler.h"
 #include "drv/hardware_targets.h"
 #include "drv/led.h"
-#include "drv/motherboard.h"
 #include "drv/timer.h"
 #include "drv/watchdog.h"
 #include "xil_printf.h"
 #include <stdbool.h>
 #include <stdio.h>
 
+#if USER_CONFIG_ENABLE_AMDS_SUPPORT == 1
+#include "drv/amds.h"
+#endif
+
```

Comment:
I do not think we need this include anymore... since we removed the code below which requests new data from the AMDS